webrtc.html

```diff
@@ -895,6 +899,20 @@
                     <var>connection</var>.</p>
                   </li>
                   <li>
+                    <p>If <var>description</var> is set as a local description,
+                    and it creates a new SCTP association, as defined in
+                    [[!SCTP-SDP]], set the value of <var>connection</var>'s
+                    [[<code><a>sctpTransport</a></code>]] internal slot
+                    to a newly created <code><a>RTCSctpTransport</a></code>.</p>
```

Comment:
Would "initiates a new SCTP association" be better wording? What I'm trying to capture is:

* data "m=" section negotiated for the first time
* data "m=" section going from rejected to non-rejected
* local port changed (which should never happen?)